.github/workflows/tests.yml

```diff
@@ -0,0 +1,13 @@
+name: Run tests
```

Comment:
```suggestion
name: Run Unit tests
```

Reply:
We can (and will) run integration tests here as well, that's why I didn't want to put `unit` in there.